docs/model_repository_template.md

```diff
@@ -19,6 +19,8 @@ This is also a pre-requisite for adding the model to the scivision "catalog", en
   - :ref:`config`
   - :ref:`pip`
   <!-- - Model adapter code (TODO: for a later version of scivision)-->
+- :ref:`requirements-for-scivision-website`
```

Comment:
Nice, this was exactly what I meant :)

Reply:
No worries at all. Glad I understood you correctly.